dali/base/danqs.cpp

```diff
@@ -531,6 +534,7 @@ class CDaliNamedQueueServer: public IDaliServer, public Thread, implements IConn
             try {
                 // 1) Need to increment nq_requests here
                 // NB: it will never be decremented. This is total for life of this instance.
+                pNqRequestsCount->inc(1);
```

Comment:
@jakesmith I think this is in the wrong place - this is before the message has been received, so will generally be too high.